app/domains.js

```diff
@@ -27,6 +27,7 @@ var approvedDomains = [
   'hs2.org.uk',
   'jncc.gov.uk',
   'judicialappointments.digital',
+  'leadershipacademy.nhs.uk',
```

Comment:
Why not just `nhs.uk` – we already have `nhs.net`, and a couple of other `.nhs.uk` domains in there that could be subsumed?

Reply:
Quite a few nhs.uk domains are delegated to external suppliers and generally people should be using nhs.net mail anyway (although we all know that doesn't hold true in a lot of cases). I'd argue it's safer to whitelist individual nhs.uk addresses for those reasons.